tools/e2e-commons/config/playwright.config.js

```diff
@@ -0,0 +1,3 @@
+const basePlaywrightConfig = require( './playwright.config.default' );
```

Comment:
How confusing this can get in time... 😆 , having both configs here. But I don't have a better solution. 
The cleanest would be to have another project depending on `e2e-commons`, how it was thought in the first place, but maybe it's an overhead, given the fact that it will only contain some config files.

Reply:
haha, yeah. as an option, we could rename the default config to `playwright.config.js`